### PR TITLE
Integrate Marathon 1.5 🏃 

### PR DESCRIPTION
**Pull request by orlandohohmeier**
---
⚠️  _Apparently there is a GH bug thus this PR integrates the changes from #2178._

---

This is an integration branch that contains Marathon 1.5 compatible changes:

✅  new Networking API #2166 
✅  Multi Network Form Warning #2167 
✅  UCR Improvements #2098 

Please use https://github.com/dcos/dcos/pull/1505 for testing.

### CODE REVIEW

**Comment by orlandohohmeier** [comment]
I fixed the tests... Let's merge this one, once it's green.

**Comment by MatApple** [comment]
When opening service create modal I get -

**Comment by nLight** [comment]
Extracted two first commits to a separate branch thus changed the base of this PR to #2213

**Comment by nLight** [comment]
#2213 is merged, so that one targets `master` now

**Comment by nLight** [comment]
retest this please

**Comment by weblancaster** [comment]
@orlandohohmeier I noticed you removed the `hold merge` label, if this is ready for review please rebase since there is a conflict. thanks.

**Comment by orlandohohmeier** [comment]
@nLight @Poltergeist There seems to be an issue with the container reducers. Could one of you please look into it?

**Comment by nLight** [comment]
@orlandohohmeier on it

**Comment by nLight** [comment]
@orlandohohmeier done, replaced all the index values with `null` either a rebase issue or they've just slipped away during the applyPatch removal efforts.

**Comment by rcorral** [comment]
I think the new RAML version is causing this weird validation behavior
gif

**Comment by MatApple** [comment]
I'm also having some validation issues. After creating an app and editing it after it successfully deploys - I get the following validation error which prevents me from ever editing it.

File: tests/pages/services/ServiceFormModal-cy.js [hunk 1]
```diff
@@ -809,12 +809,10 @@ describe("Service Form Modal", function() {
           });
         });
 
-        it('should disable the "Advanced Settings" section', function() {
-          cy.contains("Advanced Settings").parent().within(function() {
-            cy
-              .get("label.form-control-toggle.disabled")
-              .should("have.length", 2);
-          });
+        it('should disable the "Grant Runtime Privileges" checkbox', function() {
```
Comment (weblancaster) [inline tests/pages/services/ServiceFormModal-cy.js]: I believe this should say `should be disabled` instead `should disable` which make you think the test is an action to disable but it's only confirming.

Comment (orlandohohmeier) [inline tests/pages/services/ServiceFormModal-cy.js]: We should actually change the test case description to `it disables the ...` following betterspecs.org but this shouldn't block the PR.

**Comment by Poltergeist** [comment]
It is fixed now!

**Comment by orlandohohmeier** [comment]
Many thanks everyone for the hard work and the reviews. Great Job! ⚒ 💯  I'm going to merge this one despite an insufficient number of reviews as this is an integration branch and all adjustments and features have been reviewed in their respective PRs.

**Pull request merged**